### PR TITLE
[PLT-6740] Fix position of emoji picker

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix position of emoji picker at:
- center post list
- center create post
- RHS root post
- RHS root comment
- RHS create comment

#### Ticket Link
Jira ticket: 
- [PLT-6740](https://mattermost.atlassian.net/browse/PLT-6740)
- [PLT-6456](https://mattermost.atlassian.net/browse/PLT-6456)

#### Checklist
- [x] All new/modified APIs include changes to the drivers
- [x] Has UI changes

### CODE REVIEW

**Comment by jasonblais** [comment]
@lindalumitchell Can you help give this a test run, given we're a day from code complete?

**Comment by saturninoabril** [comment]
@jasonblais @lindalumitchell I've found issue - that is when the emojipicker at create comment box is positioned at the left side of the smile-icon (that is when there's no enough space on top and bottom) which I'll fix today. Kindly proceed with your testing and let me know your other observation/findings. thanks!

**Comment by lindalumitchell** [comment]
@jasonblais @saturninoabril I tested and emoji picker position looks good (chatted with Saturn and repro'ed the issue he mentioned: with short window height, click the emoji picker on a reply text box, then mouse over it to try to pick an emoji. Picker position moves down the screen). 

No other issues found.

File: webapp/components/create_comment.jsx [hunk 1]
```diff
@@ -548,15 +558,38 @@ export default class CreateComment extends React.Component {
...
+                onFileUpload={this.handleFileUploadComplete}
+                onUploadError={this.handleUploadError}
+                postType='comment'
+                channelId={this.props.channelId}
+            />
+        );
+
         let emojiPicker = null;
-        if (this.state.showEmojiPicker) {
+        if (Utils.isFeatureEnabled(Constants.PRE_RELEASE_FEATURES.EMOJI_PICKER_PREVIEW)) {
```
Comment (hmhealey) [inline webapp/components/create_comment.jsx]: Accidentally reverted line

File: webapp/components/create_post.jsx [hunk 2]
```diff
@@ -693,23 +702,46 @@ export default class CreatePost extends React.Component {
...
+                onFileUpload={this.handleFileUploadComplete}
+                onUploadError={this.handleUploadError}
+                postType='post'
+                channelId=''
+            />
+        );
+
         let emojiPicker = null;
-        if (this.state.showEmojiPicker) {
+        if (Utils.isFeatureEnabled(Constants.PRE_RELEASE_FEATURES.EMOJI_PICKER_PREVIEW)) {
```
Comment (hmhealey) [inline webapp/components/create_post.jsx]: Another accidental revert

**Comment by lindalumitchell** [comment]
Note: Created https://mattermost.atlassian.net/browse/PLT-6999 for the issue Saturnino described.

**Pull request merged**